cmd/nerdctl/start.go

```diff
@@ -95,13 +101,50 @@ func startContainer(ctx context.Context, container containerd.Container) error {
 			logrus.WithError(err).Debug("failed to delete old task")
 		}
 	}
+	flagT, err := getFlagT(ctx, container)
+	if err != nil {
+		return err
+	}
+	if flagT {
+		var con console.Console
+		con = console.Current()
+		defer con.Reset()
+		if err := con.SetRaw(); err != nil {
+			return fmt.Errorf("failed to SetRaw,err is: %w", err)
+		}
+		if con == nil {
+			return errors.New("got nil con with flagT=true")
+		}
+		taskCIO = cio.NewCreator(cio.WithStreams(con, con, nil), cio.WithTerminal)
+	}
```

Comment:
For consistency with Docker, we have to detach tty

Reply:
what should I do?   change from   `return errors.New("got nil con with flagT=true")` to `return errors.New("got nil con")`?

Reply:
Probably you should attach a dummy terminal, not the real terminal, to emulate Docker's behavior.
Needs further investigation.